### DISCUSSION TITLE
RTA-TB Implementation

### DISCUSSION THREAD

**Discussion by eandis** [gottagofaster236/SonicVisualSplit#5] [Ideas]
Would it be possible to implement rta-tb system? Would be pretty helpful for s3k runners.

**Comment by gottagofaster236** [comment]
> Would it be possible to implement rta-tb system? Would be pretty helpful for s3k runners.

Hello! I think it would possible, but it would take a lot of effort. Right now I just don't have the motivation for that, but maybe someday 🙂

**Comment by gottagofaster236** [comment]
I [did try](https://github.com/gottagofaster236/SonicVisualSplit/commit/b04bf0ee36f491443b6a90c931d2492509a51698#diff-91410f954c24ab36e72deba50f8a00b8aa3daf9248b72a1142012afde33e8632) to make it work with Sonic 3 before, but the current algorithm for OCR didn't work in Angel Island well enough. Btw, Sonic 3's "TIME" label is changed during boss fights, so the algorithm would have to be tweaked to search for two different "TIME" images.

**Comment by gottagofaster236** [comment]
Okay so there's RTA-TB for S1 and S2
